collector/lib/ConnTracker.h

```diff
@@ -138,6 +138,20 @@ class ConnectionTracker {
   // recent than the stored one.
   void EmplaceOrUpdateNoLock(const ContainerEndpoint& ep, ConnStatus status);
 
+  //
+  // Statistics on the number of stored connections and their rate creation.
+  //
+  struct Stats {
+    struct {
+      unsigned int public_;
+      unsigned int private_;
+    } inbound, outbound;
+  };
+  // Retrieve the number of connections currently known to ConnTracker, indexed by in/out and public/private nature.
```

Comment:
Would be great to add a few words about how calculation of those counters differ. E.g. that the former is getting calculated right in there, where the latter is getting updated immediately after a connection is getting added.